### PR TITLE
Set tabIndex to -1 for zoomSlider

### PR DESCRIPTION
**Pull request by edellucien**

### CODE REVIEW

**Comment by edellucien** [comment]
@oterral Preview link works on ie11.

**Comment by edellucien** [comment]
@oterral I added some ariaLabel props to some components, as well for barrierfree changes

File: src/components/BaseLayerToggler/BaseLayerToggler.js [hunk 1]
```diff
@@ -300,10 +321,18 @@ class BaseLayerToggler extends Component {
...
+            onClick={() => this.previous()}
+            title={ariaLabelPrevious}
+          >
             <FaArrowCircleLeft />
           </Button>
-          <Button className={classNameNext} onClick={() => this.next()}>
+          <Button
+            className={classNameNext}
+            onClick={() => this.next()}
+            title={ariaLabelNext}
```
Comment (oterral) [inline src/components/BaseLayerToggler/BaseLayerToggler.js]: is not really `ariaLabelNext` you set more `titleNext`

**Comment by oterral** [comment]
For aria-label, I will be consistent name all properties `titleSomething` then use the same  value in `title` attribute and `ariaLabel`.

**Comment by edellucien** [comment]
Props renamed accordingly

File: src/components/Checkbox/Checkbox.js [hunk 2]
```diff
@@ -39,6 +44,7 @@ const defaultProps = {
   onChange: () => {},
   checked: false,
   tabIndex: 0,
+  ariaLabel: null,
```
Comment (oterral) [inline src/components/Checkbox/Checkbox.js]: I guess here too we could use `title`

File: src/components/TopicList/__snapshots__/TopicList.test.js.snap [hunk 3]
```diff
@@ -575,11 +609,13 @@ exports[`Topiclist matches snapshots when classNames are used. 1`] = `
                 <span />
               </label>
               <div
+                aria-label="Lines Samples null"
                 className="tm-layer-tree-toggle"
                 onClick={[Function]}
                 onKeyPress={[Function]}
                 role="button"
                 tabIndex={0}
+                title="Lines Samples null"
```
Comment (oterral) [inline src/components/TopicList/__snapshots__/TopicList.test.js.snap]: We dont'want want `null` in the title.

**Comment by edellucien** [comment]
I defined default values for component's title.

File: src/components/LayerTree/LayerTree.js [hunk 4]
```diff
@@ -183,6 +211,9 @@ class LayerTree extends Component {
         tabIndex={tabIndex}
         inputType={inputType}
         checked={layer.getVisible()}
+        ariaLabel={
```
Comment (oterral) [inline src/components/LayerTree/LayerTree.js]:
```suggestion
        title={
```

**Pull request merged**